### PR TITLE
Update README and example notebooks with new Structured profile list paradigm (part 3/3)

### PR DESCRIPTION
**Pull request by ChrisWallace2020**
Update README and example notebooks to reflect the refactored state of the structured profile and report, that they are now lists instead of dicts. After this PR this branch will be ready to be merged into main.

### CODE REVIEW

File: .github/workflows/test-python-package.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ name: Test Python Package
 
 on:
   pull_request:
-    branches: [ main, duplicate_column_fix ]
```
Comment (ChrisWallace2020) [inline .github/workflows/test-python-package.yml:8]: Don't need to run tests on this branch anymore, just a documentation update

File: README.md [hunk 2]
```diff
@@ -101,20 +104,20 @@ The format for a structured profile is below:
...
-	    },
+                'min': int,
+                'max': int,
+                'mean': float,
+                'var': float,
+                'std': float,
+                'sample_size': int,
+                'margin_of_error': float,
+                'confidence_level': float		
+            },
```
Comment (ChrisWallace2020) [inline README.md:115]: The formatting when viewing the md file was not readable in terms of indenting, I made it match the rendered README

File: README.md [hunk 3]
```diff
@@ -60,15 +60,18 @@ The format for a structured profile is below:
     "duplicate_row_count": int,
     "file_type": string,
     "encoding": string,
+    "profile_schema": {
+        string: list(int)
+    }
```
Comment (ChrisWallace2020) [inline README.md:65]: New addition in previous PRs

File: README.md [hunk 4]
```diff
@@ -60,15 +60,18 @@ The format for a structured profile is below:
...
 },
-"data_stats": {
-    <column name>: {
+"data_stats": [
+    {
```
Comment (ChrisWallace2020) [inline README.md:68]: data_stats is a list instead of a dict now

File: examples/labeler.ipynb [hunk 5]
```diff
@@ -170,9 +170,9 @@
     "def get_structured_results(results):\n",
     "    columns = []\n",
     "    predictions = []\n",
-    "    for col in results['data_stats']:\n",
-    "        columns.append(col)\n",
-    "        predictions.append(results['data_stats'][col]['data_label'])\n",
+    "    for col_report in results['data_stats']:\n",
+    "        columns.append(col_report.column_name)\n",
```
Comment (AnhTruong) [inline examples/labeler.ipynb:174]: can we do `col_report["column_name"]` ?

Comment (ChrisWallace2020) [reply examples/labeler.ipynb:174]: Oh whoops, thats what I meant actually

Comment (ChrisWallace2020) [reply examples/labeler.ipynb]: Fixed

File: examples/unstructured_profilers.ipynb [hunk 6]
```diff
@@ -179,7 +179,7 @@
     "profile_options = dp.ProfilerOptions()\n",
     "\n",
     "# Setting multiple options via set\n",
-    "profile_options.set({ \"vocab.is_enabled\": False, \"is_case_sensitive\": True })\n",
+    "profile_options.set({ \"*.vocab.is_enabled\": False, \"*.is_case_sensitive\": True })\n",
```
Comment (ChrisWallace2020) [inline examples/unstructured_profilers.ipynb:182]: Bug in notebook from change made in previous PR where options no longer search unless specified with `*.`

**Review by JGSweets — APPROVED** [review]

**Review by AnhTruong — APPROVED** [review]

**Pull request merged**